pdns/dnsdistdist/m4/pdns_check_re2.m4

```diff
@@ -0,0 +1 @@
+../../../m4/pdns_check_re2.m4
```

Comment:
that's a feature I feel should go in general, it's a maintenance pain - and now our auth tarball knows about re2!